### DISCUSSION TITLE
Alerts hosted in a user-friendly format

### DISCUSSION THREAD

**Discussion by rosenzweigv** [weather-gov/api#656] [Q&A] [answered]
We're working at Leon County to automatically post NWS alerts for our area to social media, and have been using the `/alerts/active/zone/{zoneId}` endpoint to get the alerts. However, the project requires us to add a link in the posts to the official NWS posting of the alert.

However, as far as we can tell, there is not such location accessible from the information the API provides. We'd like to link directly to the [alerts.weather.gov](alerts.weather.gov) text for the alert, but it seems that those CAP V1.1 IDs are disjoint from the CAP V1.2 IDs that the Web Service API provides. As an example, [this is what we would like to link to](https://alerts.weather.gov/cap/wwacapget.php?x=TX126660080588.ExcessiveHeatWarning.126660160520TX.FWDNPWFWD.429a2b7ba0acbc006aaccfd133d3b452), but we cannot figure out where the URL comes from (specifically, the ID of `TX126660080588.ExcessiveHeatWarning.126660160520TX.FWDNPWFWD.429a2b7ba0acbc006aaccfd133d3b452`).

Is there some way to convert the ID from the web API to that ID? Additionally, that site is not mobile friendly, so is there a different version of [alerts.weather.gov](alerts.weather.gov) that is mobile friendly and uses the CAP V1.2 that this API seems to provide?

**Comment by rosenzweigv** [comment]
Accidentally closed, reopening

**Comment by sullynole** [comment, marked as answer]
alerts.weather.gov is currently being updated to use api v1.2 alerts.  For now, use https://alerts-v2.weather.gov

**Reply by rosenzweigv** [reply]
Thank you so much! Will alerts-v2.weather.gov remain after alerts.weather.gov is updated?

**Reply by sullynole** [reply]
Here is more info on the plan.  https://www.weather.gov/media/notification/pdfs/scn17-35cap_aaa.pdf

Will likely need to switch to alerts.weather.gov when that happens.  There will be another SCN posted here when it gets close: https://www.weather.gov/notification/#scn

**Reply by scadergit** [reply]
Note that alerts-v2 is currently be improved before it becomes alerts, and the url path and parameters will change slightly from it's current version.
